appendix.ru.md

```diff
@@ -2,23 +2,23 @@
 lang: ru
 ---
 
-Ричард М. Столлман, известный своим негативным отношением к женщинами, заставлял их чувствовать себя некомфортно и уязвимо. Информацию об инцидентах, связанных с Ричардом М. Столлманом и MIT, см. в [0].
+Ричард М. Столлман, известный своим неподобающим поведением в отношении к женщинам, заставлял их чувствовать себя некомфортно и уязвимо. Информацию об инцидентах, связанных с Ричардом М. Столлманом и MIT, см. в [0].
 
 [0]: https://selamjie.medium.com/remove-richard-stallman-appendix-a-a7e41e784f88
 
-Взгляды Ричарда М. Столлмана на законы об изнасиловании и педофилии были публично обсуждены осенью 2019 года, когда о них написала Селам Ж. [1] Особенно пугает то, как Столлман относится к обвинениям в том, что Марвин Минский изнасиловал одну из жертв торговли людьми Джеффри Эпштейна (Вирджиния Джюффре), сказав: «но наиболее вероятный сценарий событий состоит в том, что она представила себя полностью готовой к интимным связям»  [2]. Ричард М. Столлман заявляет, что это не «сексуальное насилие», потому что «нападение предполагает, что нападающий применил силу или принуждение», в то время как в обсуждаемом отчете «нет ничего подобного. Только то, что у них был секс.» [3] Вместо того, чтобы обсуждать это дальше, лучше сосредоточиться на его личном веб-сайте, где он делится своими взглядами на то, что несовершеннолетние «полностью готовы к интимным отношениям». [4]. (Примечание. Хотя несколько новостных сообщений и искажали позицию Ричарда М. Столлмана на тему обсуждения обвинений против Марвина, Ричард М. Столлман ранее сам высказывал мнение, которое полностью соответствовало искаженному.)
+Взгляды Ричарда М. Столлмана на законы об изнасиловании и педофилии были публично обсуждены осенью 2019 года, когда о них написала Селам Ж. [1] Особенно пугает то, как Столлман реагирует на обвинения Марвина Минского в изнасиловании одной из жертв торговли людьми Джеффри Эпштейна (Вирджиния Джюффре), сказав: «…но наиболее вероятный сценарий событий состоит в том, что она представила себя как полностью добровольно готовой к интимным связям»  [2]. Ричард М. Столлман заявляет, что это не «сексуальное насилие», потому что «насилие предполагает, что нападающий применил силу или принуждение», в то время как в обсуждаемом отчете «нет ничего подобного. Только то, что у них был секс.» [3] Вместо того, чтобы обсуждать это дальше, лучше сосредоточиться на его личном веб-сайте, где он делится своими взглядами на то, что несовершеннолетние «полностью добровольно готовы к интимным отношениям». [4]. (Примечание. Хотя несколько новостных сообщений и искажали позицию Ричарда М. Столлмана на тему обсуждения обвинений против Марвина, Ричард М. Столлман ранее сам высказывал мнение, которое полностью соответствовало искаженному.)
```

Comment:
1. *addresses the accusations* is probably better said *reacts to* rather than *has attitude towards*.
2. *entirely willing* was translated as *entirely ready*, my proposal is *entirely willingly ready* which is closer to the original.
3. *assault* is translated as *violence/force* in one case and *assault* in another, I propose to use the same term in both cases.